src/pages/delivery/Map.jsx

```
@@ -1,13 +1,14 @@
 import { Link, useNavigate } from "react-router-dom";
 import useMyLocation from "../../hooks/useMyLocation";
 import useFetchOrders from "../../hooks/useFetchOrders";
-import { i } from "motion/react-client";
+import BasicMap from "../../components/common/BasicMap";
+import { MapMarker } from "react-kakao-maps-sdk";
 import React, { useEffect, useCallback, useRef } from "react";
 
 export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)